configdb/settings.py

```diff
@@ -150,6 +151,7 @@ def get_list_from_env(variable, default=None):
     'DEFAULT_AUTHENTICATION_CLASSES': (
         # Allows authentication against DRF authtoken and then Oauth Server's api_token
         'ocs_authentication.backends.OCSTokenAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
```

Comment:
And this is necessary for the site-proxy to work with basic user/pass auth like it used to. I'm planning to update site-proxy to use token auth at some point and then this shouldn't be needed anymore.